### ISSUE TITLE
JS fiddle (cluster operational maps) check for Rami (check with Pedro)

### ISSUE THREAD

**Issue by javedyousafzai** [humanitarianresponse/site#103]
If I go to the cluster operational map, https://www.humanitarianresponse.info/en/coordination/clusters/countries, and select only Education clusters, and then click on the 'hamburger' icon to export the map for embedding "embed this map", I am taken to the [jsfiddle page] (http://jsfiddle.net/guillaumev/32ouwf76/). 

Now my question is, can UNICEF get only the 'Education' cluster  geojson so they can embed only Education instance in their interactive map?

This might be working but I wanted to clarify and also wanted to document it for external clients such as UNICEF, so we tell them how to use this. I would like to write some easy guidelines in our [Help site] (https://www.humanitarianresponse.info/help) 

@pcambra , I am tagging @guillaumev as well as he has done this work before.

**Comment by ramyzaki** [comment]
@pcambra 
Any updates?

**Comment by MartinMaritim** [comment]
@ramyzaki I will look into it today, sorry I didn't get time to yesterday ...

**Comment by ramyzaki** [comment]
@MartinMaritim Thanks in advance

**Comment by MartinMaritim** [comment]
I might be wrong, but it seems the jsfiddle page doesn't load with anything pre filtered. I am still looking into it and will provide some clear answers ...

**Comment by ramyzaki** [comment]
Exactly, you are not wrong, the jsfiddle page does not load the filtered query, it loads only the exact map without filter.

I think, this was not the case before, that is why it stopped working on the Global cluster websites.

**Comment by javedyousafzai** [comment]
@ramyzaki , did you check with your developer as well on why the map is not showing anything?

**Comment by ramyzaki** [comment]
He is not available right now, but I have full access to the FTP site.
But I think, the problem is with the jsfiddle, because in the export page, the map is shown with no filters.

**Comment by javedyousafzai** [comment]
@MartinMaritim will write instructions with working example on 'How to use JDFiffle' to filter out the required clusters/cluster when they export the map values.
